tests/publiccloud/transfer_repos.pm

```diff
@@ -22,10 +22,12 @@ sub run {
     my ($self, $args) = @_;
     select_console 'tunnel-console';
 
+    $args->{my_instance}->run_ssh_command(cmd => "sudo SUSEConnect -r " . get_required_var('SCC_REGCODE'), timeout => 180) unless (get_var('FLAVOR') =~ 'On-Demand');
+
     assert_script_run("rsync -va -e ssh ~/repos root@" . $args->{my_instance}->public_ip . ":'/tmp/repos'", timeout => 900);
     $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec sed -i 's,http://,/tmp/repos/repos/,g' '{}' \\;");
-    $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec zypper ar '{}' \\;");
     $args->{my_instance}->run_ssh_command(cmd => "sudo find /tmp/repos/ -name *.repo -exec echo '{}' \\;");
+    $args->{my_instance}->run_ssh_command(cmd => "sudo ls /tmp/repos/*.repo");
```

Comment:
I think you replaced the wrong line. Now you "echo" the repos twice but don't call `zypper ar`